packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -185,9 +185,9 @@ export class I18nUtils {
     if (typeof stringToFormat !== 'string')
       throw new Error("'stringToFormat' must be a String");
     return stringToFormat.replace(/{{(\d+)}}/gm, (match, index) =>
-      (formattingArguments[index] === undefined
+      formattingArguments[index] === undefined
         ? ''
-        : `${formattingArguments[index]}`)
+        : `${formattingArguments[index]}`
```

Comment:
this will not work with eslint xD

Reply:
Then I'll exclude js from prettier

Reply:
Going to try to sync eslint and prettier then